src/thread-utils.h

```diff
@@ -235,37 +235,38 @@ GIT_INLINE(int64_t) git_atomic64_get(git_atomic64 *a)
 
 #else
 
-GIT_INLINE(int) git_threads_global_init(void) { return 0; }
+GIT_INLINE(int) git___noop(void) { return 0; }
```

Comment:
couldn't think of a better name. also, lmk if we need to do something clever to explicitly mark the arguments as unused (right now it's a non-issue on my machine).